Modules/ITS/src/ITSFhrTask.cxx

```diff
@@ -629,26 +642,27 @@ void ITSFhrTask::monitorData(o2::framework::ProcessingContext& ctx)
     }
   }
 
-  //fill Occupancy plots, chip stave occupancy plots and error statistic plots
+  // fill Occupancy plots, chip stave occupancy plots and error statistic plots
   for (int i = 0; i < (int)activeStaves.size(); i++) {
     int istave = activeStaves[i];
     mOccupancyPlot[lay]->Add(occupancyPlotTmp[i]);
     if (lay < NLayerIB) {
       for (int ichip = 0; ichip < nChipsPerHic[lay]; ichip++) {
-        mChipStaveOccupancy[lay]->SetBinContent(ichip + 1, istave + 1, mOccupancy[istave][ichip]);
+        mChipStaveOccupancy[lay]->SetBinContent(ichip + 1, istave + 1, mOccupancyLane[istave][ichip]);
         int ilink = ichip / 3;
         for (int ierror = 0; ierror < o2::itsmft::GBTLinkDecodingStat::NErrorsDefined; ierror++) {
           if (mErrorVsFeeid && (mErrorCount[istave][ilink][ierror] != 0)) {
             mErrorVsFeeid->SetBinContent(((istave + StaveBoundary[lay]) * 3) + ilink + 1, ierror + 1, mErrorCount[istave][ilink][ierror]);
           }
         }
       }
-      mGeneralOccupancy->SetBinContent(istave + 1 + StaveBoundary[mLayer], *(std::max_element(mOccupancy[istave], mOccupancy[istave] + nChipsPerHic[lay])));
+      mGeneralOccupancy->SetBinContent(istave + 1 + StaveBoundary[mLayer], *(std::max_element(mOccupancyLane[istave], mOccupancyLane[istave] + nChipsPerHic[lay] / 2)));
```

Comment:
the division by 2 at the end is not needed here because this is for Inner Barrel for which we have 9 chips.

Reply:
confirmed